### PR TITLE
[ADD] Add account_filter_paybale_receivable addons

### PR DESCRIPTION
**Pull request by adrienpeiffer**
This module was written to extend the functionality of search on journal items to allow you to filter by payable and receivable account

### CODE REVIEW

File: account_filter_payable_receivable/README.rst [hunk 1]
```diff
@@ -0,0 +1,44 @@
+account_filter_payable_receivable
```
Comment (yvaucher) [inline account_filter_payable_receivable/README.rst]: Don't put the technical name as title but the module name "Account Filter Payable Receivable"

File: account_filter_payable_receivable/README.rst [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+For further information, please visit:
+
+ * https://www.odoo.com/forum/help-1
+
+
+Credits
+=======
+
+Contributors:
```
Comment (yvaucher) [inline account_filter_payable_receivable/README.rst]: Don't forget to make this a subtitle with `------`

File: account_filter_payable_receivable/README.rst [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+* Stéphane Bidoul (ACSONE) <stephane.bidoul@acsone.eu>
+* Adrien Peiffer (ACSONE) <adrien.peiffer@acsone.eu>
+
+Maintainer:
```
Comment (yvaucher) [inline account_filter_payable_receivable/README.rst]: Same here make it a subtitle

File: account_filter_payable_receivable/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+#     GNU Affero General Public License
+#     along with account_filter_payable_receivable.
+#     If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    'name': "Account Filter Payable Receivable",
+
+    'summary': """
+        Filter your Journal Items per payable and receivable account""",
```
Comment (yvaucher) [inline account_filter_payable_receivable/__openerp__.py]: Correct me if I'm wrong but the spaces preceding `Filter` will be part of the summary

**Comment by yvaucher** [comment]
That was fast. Thanks for the changes. Apart from summary remark :+1:

File: account_filter_payable_receivable/README.rst (hunk 2 above)
Comment (yvaucher) [inline account_filter_payable_receivable/README.rst]: ok

File: account_filter_payable_receivable/README.rst (hunk 3 above)
Comment (yvaucher) [inline account_filter_payable_receivable/README.rst]: ok

File: account_filter_payable_receivable/__openerp__.py (hunk 4 above)
Comment (adrienpeiffer) [inline account_filter_payable_receivable/__openerp__.py]: @yvaucher spaces preceding the description are not displayed
cpt02

**Comment by adrienpeiffer** [comment]
@yvaucher Thanks for the review !

**Comment by pedrobaeza** [comment]
A more suitable name would be _account_move_line_payable_receivable_filter_, don't you think?

**Comment by adrienpeiffer** [comment]
Good idea @pedrobaeza !

**Comment by pedrobaeza** [comment]
:+1:

**Comment by sbidoul** [comment]
:+1:

**Comment by Cedric-Pigeon** [comment]
:+1:

**Pull request merged**